### ISSUE TITLE
ConstantSpacingSniff warning

### ISSUE THREAD

**Issue by enumag** [slevomat/coding-standard#1620]
I got this warning:

```
Warning: Undefined array key "" in /app/vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/Classes/ConstantSpacingSniff.php on line 61

Warning: Trying to access array offset on value of type null in /app/vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/Classes/ConstantSpacingSniff.php on line 61
```

It seems to be triggered by an enum class with a const. In my case it was something like this:

```php
enum A: string
{
    public const X = [
        self::A,
        self::B,
    ];

    case A = 'a';

    case B = 'b';
}
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/a3c437f22cdcc7c07f3f178484cb8c3c097570ac

**Comment by lptn** [comment]
Hey @kukulich 
I have the same problem for the following code:
```php
<?php

namespace App;

enum EventCategory: string
{
    public const CATEGORY_SECURITY = 1;

    case FOO = 'BAR';
}
```
```
1 | ERROR | [ ] An error occurred during processing; checking has been aborted. The error message was: Undefined array
   |       |     key "" in
   |       |     /Users/alies/code/IxDF/IxDF-web/vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/Classes/ConstantSpacingSniff.php
   |       |     on line 62 (Internal.Exception)

```
